scripts/runlogwatch.sh

```diff
@@ -1,20 +1,24 @@
+
 #!/usr/bin/bash
 
 # Ramdisk logs path
-LOG_DIR="/shared/log/ironic/deploy"
+LOG_DIRS=("/shared/log/ironic/deploy" "/shared/log/ironic-inspector/ramdisk")
 
-while :
-do
-    until  ls "${LOG_DIR}"/*.tar.gz 1> /dev/null 2>&1
-    do
-        sleep 5
-    done
+for LOG_DIR in ${LOG_DIRS[@]}; do
+    if [[ -d $LOG_DIR ]]; then
+        while :
+        do
+            until  ls "${LOG_DIR}"/*.tar.gz 1> /dev/null 2>&1
+            do
+                sleep 5
```

Comment:
We could potentially get stuck in this loop checking one directory and not swapping back to check the other.
e.g. if we are just redeploying nodes that have been previously inspected. Then this will get stuck waiting for a file
in /shared/log/ironic-inspector/ramdisk and never return to /shared/log/ironic/deploy

Reply:
I realize that, maybe we can do it as a follow-up?
we need to fix it first in https://github.com/metal3-io/ironic-image/blob/master/scripts/runlogwatch.sh

Reply:
ack,